### PR TITLE
Update addon-templates-v1.19.8.md

### PR DESCRIPTION
**Pull request by DanielHabenicht**
## Summary of the Pull Request

add link to documentation

## PR Checklist

* [ ] Refers to: #xxx
* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.
* [ ] Release notes added.

## Acceptance Steps Performed

```
=== RUN   TestAccSomethingV0_basic
--- PASS: TestAccSomethingV0_basic (101.71s)
=== RUN   TestAccSomethingV0_timeout
--- PASS: TestAccSomethingV0_timeout (128.67s)
PASS

Process finished with exit code 0
```

### CODE REVIEW

**Comment by lego963** [comment]
@DanielHabenicht can you put link after `autoscaling` title? line 23

PS: It would be perfect if you put links for each addon from usermanual

File: opentelekomcloud/services/cce/addon-templates-v1.19.8.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ are no unneeded nodes.
 }
 ```
 
+See the [OTC Cloud Container Enging Addon Documentation](https://docs.otc.t-systems.com/en-us/usermanual2/cce/cce_01_0154.html) for more information.
```
Comment (lego963) [inline opentelekomcloud/services/cce/addon-templates-v1.19.8.md:42]: Cloud Container Engine Addon

Comment (lego963) [inline opentelekomcloud/services/cce/addon-templates-v1.19.8.md]: Cloud Container Engine

**Review by lego963 — CHANGES_REQUESTED** [review]

File: opentelekomcloud/services/cce/addon-templates-v1.19.8.md (hunk 1 above)
Comment (DanielHabenicht) [reply opentelekomcloud/services/cce/addon-templates-v1.19.8.md]: Sure, I will fix it tomorrow with the other suggestion

Comment (lego963) [reply opentelekomcloud/services/cce/addon-templates-v1.19.8.md]: @DanielHabenicht will you continue here? :)

Comment (DanielHabenicht) [reply opentelekomcloud/services/cce/addon-templates-v1.19.8.md]: thanks for the reminder.

**Review by lego963 — APPROVED** [review]

**Pull request merged**